supply/lib/supply/uploader.rb

```diff
@@ -152,11 +152,17 @@ def upload_mapping(apk_version_codes)
     end
 
     def upload_bundles
-      aab_path = Supply.config[:aab]
-      return [] unless aab_path
+      aab_paths = [Supply.config[:aab]] unless (aab_paths = Supply.config[:aab_paths])
+      return [] unless aab_paths.compact
```

Comment:
Ah sorry, I was looking at how this was implemented in the `upload_apks` - https://github.com/fastlane/fastlane/pull/13543/files#diff-fd047a79b6e303ba583ff34a4c489d78L134. `upload_apks` doesn't early return so doing a `aab_paths.compact!` on its own lined would worked.

I think it will always get passed with `return [] unless aab_paths.compact` (as long as `aab_paths` isn't `nil`). If `aab_paths` is `nil` this looks like it will 💥 .

I think what we actually want is something like...

```suggestion
      return [] unless aab_paths
      aab_paths.compact!
```

This way it will early return with an empty array if `aab_paths` is `nil`, otherwise it will remove all possible `nil` files inside of `aab_paths`